alerting/src/test/java/org/hawkular/metrics/alerting/PublishDataPointsTest.java

```diff
@@ -62,7 +61,7 @@
 import rx.Observable;
 
 /**
- * TODO: We need to mock out AlertsService, I think, so that InsertedDataSubscriber can instantiate correctly
+ * Note that this test no longer tests filtering, as data is now filtered in alerting, not metrics.
```

Comment:
well, it minimally can just ensure we're sending with a valid prefix.  I'll leave it.  there will also be an itest for ear deployment and alert generation.  the alert generation may be a follow-up PR.